core/templates/dev/head/components/loading/LoadingDotsDirective.ts

```diff
@@ -20,7 +20,11 @@ oppia.directive('loadingDots', [
   'UrlInterpolationService', function(UrlInterpolationService) {
     return {
       restrict: 'E',
+      scope: {},
+      bindToController: {},
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-        '/components/loading/loading_dots_directive.html')
+        '/components/loading/loading_dots_directive.html'),
+      controllerAs: '$ctrl',
```

Comment:
Just to check -- in cases like this, do we even need the controllerAs and controller fields?

Reply:
Yes, it is mandatory for any component directive to have all these fields as specified in the guide [here](https://angular.io/guide/upgrade#using-component-directives).